tests/rptest/scale_tests/tiered_storage_stability_test.py

```diff
@@ -213,27 +220,12 @@ def test_with_cloud_storage(self):
         Make segments replicate to the cloud, then disrupt S3 connectivity
         and restore it
         """
-        config = {
+        self.setup_cluster(
             # Segments should go into the cloud at a reasonable rate,
             # that's why it is smaller than it should be
-            'segment.bytes': int(self.scaled_segment_size/2),
-
-            # Use infinite retention so there aren't sudden, drastic,
-            # unrealistic GCing of logs.
-            'retention.bytes': -1,
-
-            # Keep the local retention low for now so we don't get bogged down
-            # with an inordinate number of local segments.
-            'retention.local.target.bytes': 2 * self.scaled_segment_size,
-            'cleanup.policy': 'delete',
-            'partition_autobalancing_node_availability_timeout_sec': self.unavailable_timeout,
-            'partition_autobalancing_mode': 'continuous',
-            'raft_learner_recovery_rate': 10 * 1024 * 1024 * 1024,
-        }
-        self.rpk.create_topic(self.topic_name,
-                              partitions=self.scaled_num_partitions,
-                              replicas=3,
-                              config=config)
+            segment_bytes = int(self.scaled_segment_size/2),
+            retention_local_bytes=2 * self.scaled_segment_size,
```

Comment:
nit: perhaps this test case should have a separate, unrelated segment size to avoid confusion over why we're scaling down

Reply:
still, for lower tiers in tiered tests we would probably want smaller segment sizes - to keep the rate, and if so it makes sense to tie it to `scaled_segment_size`